File: samples/dotNET/vcs2010/ReportsSampleApplication/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using Field;

namespace ReportsSampleApplication
{
    public partial class Form1 : Form
    {
        private string paramTemplate = "{{" +
"    \"template\": {{\"paper\": \"A4\"}}," +
"    \"context\": {{" +
"        \"hello\": {{" +
"            \"new\": \"Tx\"," +
"            \"value\": \"{0}\"," +
"            \"rect\": [100, 700, 400, 750]" +
"        }}" +
"    }}" +
"}}";

        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            var reports = new Reports();
            var param = String.Format(paramTemplate, textBox1.Text);
            reports.Render(param, textBox2.Text);
        }
    }
}

File: examples/dotNET/vb2010/ReportsSampleApplication/Form1.vb
Imports Field

Public Class Form1

    Private paramTemplate = "{{" & _
"    ""template"": {{""paper"": ""A4""}}," & _
"    ""context"": {{" & _
"        ""hello"": {{" & _
"            ""new"": ""Tx""," & _
"            ""value"": ""{0}""," & _
"            ""rect"": [100, 700, 400, 750]" & _
"        }}" & _
"    }}" & _
"}}"

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Dim reports As New Reports
        Dim param As String = String.Format(paramTemplate, TextBox1.Text)
        reports.Render(param, TextBox2.Text)
    End Sub

End Class
